.gitignore

```
@@ -24,3 +24,5 @@ web/docs
 .settings/
 debug.log
 .vscode/settings.json
+.idea/
+yarn.lock
```

please remove this and use npm for this project
```suggestion
yarn.lock
```

---

done!!